hack/targetgen/scaffolding/config/301-newtarget.yaml

```diff
@@ -0,0 +1,122 @@
+# Copyright 2021 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: {{.Kind}}s.targets.triggermesh.io
```

Comment:
I didn't realize before reading this but `Kind` and `TitleCase` seem to be used wrongly across the code.
Ref. to my previous comment: https://github.com/triggermesh/triggermesh/pull/253#discussion_r745583399

Kind is `MyCustomKind` as per Kubernetes definition (check the `kind:` field in any YAML).

We should use proper meaning for those things.
`Kind` is be what you currently call `TitleCase` (and what the user passes using a flag).